gpii/node_modules/lifecycleManager/test/js/LifecycleManagerQueueTests.js

```diff
@@ -0,0 +1,128 @@
+/*!
+GPII Lifecycle Manager Queue Tests
+
+Copyright 2017 Raising The Floor - International
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+/* eslint-env browser */
+/* eslint strict: ["error", "function"] */
+
+/* global jqUnit, fluid */
+
+(function () {
+    "use strict";
+    fluid.setLogging(true);
+
+    var gpii = fluid.registerNamespace("gpii");
+
+    fluid.registerNamespace("gpii.tests.lifecycleManager.queue");
+
+    var promiseWrapFunc = function (timeout, func, reject) {
```

Comment:
This function has an unfortunate name, and should better reflect its purpose. Also, needs doc-style comments